README.md

```diff
@@ -4,17 +4,17 @@ Implementations of MakerDAO surplus auctions, triggered on `vow.flap`.
 
 ### Splitter
 
-Exposes a `kick` operation to be triggered periodically. Its logic withdraws `DAI` from the `vow` and splits it in two parts. The first part (`burn`) is sent to the underlying `flapper` contract to be processed by the burn engine. The second part (`WAD - burn`) is distributed as reward to a `farm` contract. The `kick` cadence is determined by the `hop` value.
+Exposes a `kick` operation to be triggered periodically. Its logic withdraws `NST` from the `vow` and splits it in two parts. The first part (`burn`) is sent to the underlying `flapper` contract to be processed by the burn engine. The second part (`WAD - burn`) is distributed as reward to a `farm` contract. The `kick` cadence is determined by the `hop` value.
```

Comment:
Should we use `vat.dai` here? 
```suggestion
Exposes a `kick` operation to be triggered periodically. Its logic withdraws `vat.dai` from the `vow`, converts it into `NST`, and splits it in two parts. The first part (`burn`) is sent to the underlying `flapper` contract to be processed by the burn engine. The second part (`WAD - burn`) is distributed as reward to a `farm` contract. The `kick` cadence is determined by the `hop` value.
```

Reply:
Maybe better to keep it simple, as you did.

Reply:
Both seem correct to me. As it is true that initiates with a `vat.dai` move, but then exists to `NST` (as before it did to `DAI`). So I would probably slightly tend to leave it with `NST`.

Reply:
Ok, let's keep it as-is, resolving.